src/scss/grommet-core/_objects.form-field.scss

```diff
@@ -78,13 +78,39 @@ $form-vertical-padding: quarter($inuit-base-spacing-unit);
 
     > select {
       display: block;
-      background-position: center right $form-horizontal-padding;
+      // TODO figure out how to make this consistent with intuit and not have magic numbers
+      background-position: center right #{$form-horizontal-padding - 6};
+      padding-left: #{$form-horizontal-padding - 2};
```

Comment:
Yeah. Each browser does their own different rendering of whitespace/padding. Usually in the past to make it consistent I've had to sadly not use select boxes at all.